src/tests/ftest/recovery/container_cleanup.py

```diff
@@ -0,0 +1,129 @@
+        :avocado: tags=ContainerCleanupTest,test_container_label_inconsistency
+        """
+        # 1. Create a pool and a container.
+        self.log_step("Create a pool and a container")
+        pool = self.get_pool(connect=False)
+        container = self.get_container(pool=pool)
+
+        # 2. Inject fault to cause container label inconsistency.
+        self.log_step("Inject fault to cause container label inconsistency.")
+        daos_command = self.get_daos_command()
+        daos_command.faults_container(
+            pool=pool.identifier, cont=container.identifier,
+            location="DAOS_CHK_CONT_BAD_LABEL")
+
+        # 3. Try to access property using the original label in CS.
+        self.log_step("Try to access property using the original label in CS.")
+        access_failed = False
+        try:
+            _ = daos_command.container_get_prop(
+                pool=pool.identifier, cont=container.identifier)
+        except CommandFailure:
+            access_failed = True
+        if not access_failed:
+            self.fail("Label inconsistency fault wasn't injected property!")
+
+        # 4. Show that PS has the original label by using new-label.
+        cont_prop = daos_command.container_get_prop(
+            pool=pool.identifier, cont="new-label", properties=["label"])
+        ps_label = cont_prop["response"][0]["value"]
+        errors = []
+        if ps_label != container.identifier:
+            msg = (f"Unexpected label in PS before repair! Expected = "
+                   f"{container.identifier}; Actual = {ps_label}")
+            errors.append(msg)
+
+        # 5. Enable the checker.
+        self.log_step("Enable the checker.")
+        dmg_command = self.get_dmg_command()
+        dmg_command.check_enable()
+
+        # 6. Set policy to --all-interactive.
+        self.log_step("Set policy to --all-interactive.")
+        dmg_command.check_set_policy(all_interactive=True)
+
+        # 7. Start the checker and query the checker until the fault is detected.
+        self.log_step("Start and query the checker until the fault is detected.")
+        seq_num = None
+        # Start checker.
+        dmg_command.check_start()
+        # Query the checker until expected number of inconsistencies are repaired.
+        for _ in range(8):
+            check_query_out = dmg_command.check_query()
+            # Status is INIT before starting the checker.
+            if check_query_out["response"]["status"] == "RUNNING" and\
+                    check_query_out["response"]["reports"]:
+                seq_num = check_query_out["response"]["reports"][0]["seq"]
+                break
+            time.sleep(5)
+        if not seq_num:
+            self.fail("Checker didn't detect any fault!")
+
+        # 8. Repair by selecting "Trust the container label in container property."
+        self.log_step(
+            "Repair by selecting \"Trust the container label in container property.\"")
```

Comment:
FYI you can use single quotes to avoid escaping
```suggestion
        self.log_step(
            'Repair by selecting "Trust the container label in container property."')
```

Reply:
Done. Thanks.